README.md

```diff
@@ -90,8 +90,9 @@ end-patch
 remove frame_1/DoAction.as 789-1111
 ```
 
-You can use \# to write comments. There are two types of commands, `add` and `remove`. The first parameter to any command is the file to modify (in this case, "DefineSprite_1058 boss2/DoAction.as" or "frame_1/DoAction.as"). To find the name of this, export all scripts using FFDec and make a note of the file name you want to modify.
+You can use \# to write comments. There are several types of commands, `add` and `remove` being two of them. The first parameter to any command is the file to modify (in this case, "DefineSprite_1058 boss2/DoAction.as" or "frame_1/DoAction.as"). To find the name of this, export all scripts using FFDec and make a note of the file name you want to modify.
```

Comment:
```suggestion
Patch files consist of a set of commands, separated onto lines. You can use \# to write comments. The first parameter to any command is the file to modify (in this case, "DefineSprite_1058 boss2/DoAction.as" or "frame_1/DoAction.as"). To find the name of this, export all scripts using FFDec and make a note of the file name you want to modify.
```
The sentence you changed here is somewhat redundant now that the commands are specified individually below. To me, it makes most sense to remove it. Doing so requires a minor rewrite for legibility, for which I have included a suggestion, but it doesn't have to be exactly as I wrote it.

Reply:
Sounds good, I'll rework this part of the README.